app/controllers/api/v1/forecast_controller.rb

```diff
@@ -0,0 +1,16 @@
+class Api::V1::ForecastController < ApplicationController
+  def index
```

Comment:
Would love to refactor this controller to make it "skinnier"
Will come back to this!